### PR TITLE
Swappable pagination buttons by configulation

### PR DESCRIPTION
**Pull request by dojineko**
~~adjust to original hexo tranquilpeak theme~~
~~ref: [hexo's demo page](http://louisbarranqueiro.github.io/hexo-theme-tranquilpeak/2015/05/28/Elements-showcase/)~~

add `swapPaginator` option to configulation.
default pagenator looks like `< NEXT | PREV >`
image

If set this value to `true`, it looks like `< PREV | NEXT >`
(it seems general UI design, hexo's one, google, amazon etc,etc...)
image

updated: 2017-09-23

### CODE REVIEW

**Comment by kakawait** [comment]
@dojineko Already discussed on past (see https://github.com/kakawait/hugo-tranquilpeak-theme/issues/118) I think I will not change it again because I will be _infinite loop_ request.

But we can imagine a configuration to enable to switch. Personally I prefer the current (this theme not Hexo original) version.

**Comment by dojineko** [comment]
@kakawait 
oops, i understood. i make it changeable by configuration.

**Comment by dojineko** [comment]
@kakawait updated! plz check this later :)

**Comment by kakawait** [comment]
Thank for contribution I will check that tomorrow, sorry for delay

File: exampleSite/config.toml [hunk 1]
```diff
@@ -172,6 +172,10 @@ canonifyurls = true
   # they have to be referred from static root. Example
   # customJS = ["js/foo.js"]
 
+  # Display `Next` on left side of the pagination, and `Prev` on right side one.
+  # If you set this value to `true`, these positions swap.
+  # swapPagenator = true
```
Comment (kakawait) [inline exampleSite/config.toml]: Do you mean `swaPaginator` instead pag**e**nator?

Comment (dojineko) [reply exampleSite/config.toml]: fixed!

**Comment by kakawait** [comment]
I was very busy sorry for delay. Thank you for your contribution

**Comment by yamila-moreno** [comment]
Hi! is this feature still working? I'm using both "true" and "false" swapPaginator but it always enters in the "else" part. I tried to print the value {{ .Site.Params.swapPaginator }} but I didn't see anything, which is expected if the value is not accessed.

On regard of this value, I suggest to change the `config.toml` commented documentation from "true" to "false", because "false" is the default value; I believe that the commented code is expected to be the default (so uncommenting that line woudn't change behaviour).

**Pull request merged**